v3/src/components/case-card/case-card.tsx

```diff
@@ -22,7 +22,7 @@ export const CaseCard = observer(function CaseCard({ setNodeRef }: IProps) {
   const { active } = useDndContext()
   const instanceId = useInstanceIdContext() || "case-card"
 */
-  const data = useDataSetContext()
+  const {data, metadata} = useDataSet()
```

Comment:
```suggestion
  const { data } = useDataSet()
```
You could also revert back to `useDataSetContext()`... I'm not sure if there's an advantage either way.